pkg/serviceaccount/legacy.go

```diff
@@ -40,7 +48,11 @@ func LegacyClaims(serviceAccount v1.ServiceAccount, secret v1.Secret) (*jwt.Clai
 		}
 }
 
-const LegacyIssuer = "kubernetes/serviceaccount"
+const (
+	LegacyIssuer          = "kubernetes/serviceaccount"
+	LastUsedAnnotationKey = "kubernetes.io/legacy-token-last-used"
+	InUseLabelKey         = "kubernetes.io/legacy-token-in-use"
```

Comment:
if we do end up adding both a label and an annotation with the same data (which I'm not sure we have a need to now), is there a reason not to use the same key for both?